app/models/article.rb

```diff
@@ -1,5 +1,7 @@
 class Article < ApplicationRecord
+    has_and_belongs_to_many :category
     validates :title, presence: true
     validates :content, presence: true
     validates :lead, presence: true
+    has_and_belongs_to_many :categories
```

Comment:
Do we need both category and categories?

Reply:
Category removed